tcms/rpc/tests/test_testexecutionstatus.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=attribute-defined-outside-init, invalid-name, objects-update-used
+
+from tcms.rpc.tests.utils import APITestCase
+from tcms.tests.factories import TestExecutionFactory
+
+
+class TestFilter(APITestCase):
```

Comment:
Missing coverage for the permissions decorator, see `rpc/tests/test_testexecution.py` and APIPermissionsTestCase.